notebooks/tel_and_site/subsys_req_ver/tma/SITCOM-1120_az_brake_analysis.ipynb

```diff
@@ -0,0 +1,581 @@
+{
```

Comment:
<p>In general, this is already super good and we can start using it. As you mention, the next step would be to collect an event and show it in the plots. </p><p>I recommend you use the <code><a href="https://ts-xml.lsst.io/sal_interfaces/MTMount.html#error" target="_blank">logevent_error</a></code> event from <code>MTMount</code> . The only information you need from it is its timestamp. You should be able to add a vertical line with <code>plt.axvline(x_position)</code> or, if you are using axes instead, <code>ax.axvline(x_position)</code> . No need to other log errors, I think.</p><p> </p><p>In addition, we are interested in learning how long did the TMA move after this event until it stopped. So we need to define a criteria that says that the telescope stopped based solely in the <code>actualPosition</code> . Maybe we can use the actual velocity too.  </p><p> </p><p>    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>